src/test/java/org/gridsuite/study/server/VoltageInitTest.java

```diff
@@ -358,13 +359,17 @@ public void testResetUuidResultWhenVoltageInitFailed() {
 
         StudyService studyService = Mockito.mock(StudyService.class);
         doAnswer(invocation -> {
-            input.send(MessageBuilder.withPayload("").setHeader(HEADER_RECEIVER, resultUuidJson).build(), voltageInitFailedDestination);
+            input.send(
+                MessageBuilder.withPayload("")
+                    .setHeader(HEADER_RECEIVER, resultUuidJson)
+                    .setHeader("resultUuid", VOLTAGE_INIT_ERROR_RESULT_UUID)
+                .build(), voltageInitFailedDestination);
             return resultUuid;
         }).when(studyService).runVoltageInit(any(), any(), any());
         studyService.runVoltageInit(studyEntity.getId(), modificationNode.getId(), "");
 
-        // Test reset uuid result in the database
-        assertTrue(networkModificationTreeService.getVoltageInitResultUuid(modificationNode.getId()).isEmpty());
+        // Test result in not reseted anymore in the database
```

Comment:
You can remove the anymore, it should be in the commit message but not the code comments

Reply:
fixed